File: Modul 2/SOAL-4/PRAK204_2410817220015_Ashma_Salimah.py
import math
r_tinggi = []
while True :
    bil = input("")
    if not bil :
        break
    r_tinggi.extend(bil.split())
r_tinggi = list(map(float, r_tinggi))
jari, tinggi = r_tinggi
volume = 22 * jari * jari * tinggi / 7
luas = 2 * 22 * jari * (jari + tinggi) / 7
keliling = 2 * 22 * jari / 7
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul 2/SOAL-4/PRAK204_2410817220015_Ashma_Salimah.c
#include <stdio.h>
#include <math.h>
int main() {
    float jari, tinggi, volume, luas, keliling;
    scanf("%f", &jari);
    scanf("%f", &tinggi);
    volume = 22 * jari * jari * tinggi / 7;
    luas = 2 * 22 * jari * (jari + tinggi) / 7;
    keliling = 2 * 22 * jari / 7;
    printf("Volume = %.2f \n", volume);
    printf("Luas = %.2f \n", luas);
    printf("Keliling = %.2f \n", keliling);
    return 0;
}